packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -117,40 +124,66 @@ export async function getGasPricesInNativeWei(
   };
 }
 
-export async function getNativeWeiInToken(provider: JsonRpcProvider, tokenAddress: string): Promise<BN> {
+export async function getNativeToTokenRate(provider: JsonRpcProvider, tokenAddress: string): Promise<TokenPairRate> {
```

Comment:
I think it would be great to add an example in a code comment above the function declaration (input and output of a concrete example). I'm noticing we sometimes reimplement the same things in our conversion utils and I believe example comments would help to reduce that